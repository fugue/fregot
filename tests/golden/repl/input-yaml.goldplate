{
    "command": "fregot",
    "arguments": ["repl", "--no-history-file"],
    "asserts": [
        {"exit_code": 0},
        {"stderr": "${GOLDPLATE_NAME}.stderr"},
        {"stdout": "${GOLDPLATE_NAME}.stdout"}
    ],
    "stdin": [
        ":input input-yaml.yml",
        "input"
    ]
}
