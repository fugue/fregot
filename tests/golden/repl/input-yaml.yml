request:
  method: GET
  path: /api/v1/users
